src/widgets/WalletModal/CopyToClipboard.tsx

```diff
@@ -30,17 +30,33 @@ const Tooltip = styled.div<{ isTooltipDisplayed: boolean }>`
 const CopyToClipboard: React.FC<Props> = ({ toCopy, children, ...props }) => {
   const [isTooltipDisplayed, setIsTooltipDisplayed] = useState(false);
 
+  const copyToClipboard = (content: string) => {
+    const el = document.createElement("textarea");
+    el.value = content;
+    document.body.appendChild(el);
```

Comment:
Was going to comment that we should specify `el.style.display = "none"` or `el.style.visibility = "hidden"` just to be safe. But - apparently `el.select()` doesn't work if the element is not actually visible on the screen.

Thankfully this happens quick enough so it is completely not noticeable for user.